dts/bindings/base/mutable.yaml

```diff
@@ -7,7 +7,7 @@ properties:
   zephyr,mutable:
     type: boolean
     description: |
-      True iff the device structure may be mutated.
+      True if the device structure may be mutated.
```

Comment:
@cfriedt was the "iff" intentional here? :)

Reply:
Yes - please replace with "if and only if"